waves/wallet/Cargo.toml

```diff
@@ -13,7 +13,7 @@ default = [ "console_error_panic_hook" ]
 [dependencies]
 aes-gcm-siv = { version = "0.9", features = [ "std" ] }
 anyhow = "1"
-bdk = { git = "https://github.com/bitcoindevkit/bdk", default-features = false }
+bdk = { git = "https://github.com/coblox/bdk", default-features = false }
```

Comment:
Could you not use `[patch.crates-io]` to replace the `miniscript` dependency?

Reply:
Ah, I forget how to use this feature. That might be better!

Reply:
You were right, one fewer fork to maintain.

Reply:
It is already [merged](https://github.com/rust-bitcoin/rust-miniscript/pull/212) into master so you could even depend on their HEAD directly instead of our fork.

Reply:
I tried that, but there's a tricky breaking change in between 4.0 (the version `bdk` depends on) and our commit.